app/page.tsx

```diff
@@ -7,8 +7,11 @@ import Button from '@mui/material/Button';
 import Grid from '@mui/material/Grid';
 import Divider from '@mui/material/Divider';
 import TextInput from '@/components/TextInput';
+import SelectInput from '@/components/SelectInput';
 import { FormValues } from '@/types';
 
+const whiskeyOptions = ['Whiskey 1', 'Whiskey 2', 'Whiskey 3'];
```

Comment:
We usually create an object for options prop with the following shape:
```ts
type Option = {
  value: string;
  label: string;
};

type Options = Array<Option>;
```

This allows us to save standardised values in the database while putting any label (for example, translation label) as label visible to a user.